syntax="proto3";

package gravity_pb;

message WindowEntity {
	float posX = 1;
	float posY = 2;
	float posZ = 3;
	float sizeX = 4;
	float sizeY = 5;
	fixed32 up_plane = 6;
	float texture_scale = 7;
	fixed32 window_type = 8;
	fixed32 water_block_mode = 10;
}
